src/Option.php

```diff
@@ -277,6 +277,10 @@ protected function _preprocessValue($value)
             }
         }
 
+        if (!$this->validate($value)[0]) {
```

Comment:
This change breaks compatibility with PHP 5.3 which is still declared in `composer.json`. @c9s do you want to bump PHP requirements or fix this issue?

Reply:
@morozov shall we use an extra variable to improve the compatibility?

Reply:
Yes, I believe it will work.